instrumentation/runtime-metrics/runtime-metrics-java8/library/src/main/java/io/opentelemetry/instrumentation/runtimemetrics/java8/Cpu.java

```diff
@@ -89,7 +89,7 @@ List<AutoCloseable> registerObservers(
         meter
             .gaugeBuilder("process.runtime.jvm.system.cpu.load_1m")
             .setDescription("Average CPU load of the whole system for the last minute")
-            .setUnit("1")
+            .setUnit("{run_queue_length}")
```

Comment:
@zeitlinger I hope it's ok, I'm going to accept this change and merge, so we can get the (more important) parts into the upcoming release, since we don't have resolution yet on https://github.com/open-telemetry/semantic-conventions/pull/95
```suggestion
            .setUnit("1")
```